backends/bmv2/pna_nic/README.md

```diff
@@ -0,0 +1,29 @@
+# BMv2 `pna_nic` Backend
```

Comment:
@AdarshRawat1 Does this documentation work with doxygen?

Reply:
No, This Markdown heading format is not yet officially released, it's a [work in progress.](https://github.com/doxygen/doxygen/issues/10466) . As of today, this produces the output:
![image](https://github.com/user-attachments/assets/6ab1efc8-c277-48b6-8cc5-d09e15050e67)

## Alternatives 
 
| Code  | GitHub preview | Doxygen Output  |
|--------|--------|--------|
 |  ``` # BMv2 <code>pna_nic</code> Backend ```| ![image](https://github.com/user-attachments/assets/72911b2a-fbcb-44f1-80ef-b7ad3c374ca7) | ![image](https://github.com/user-attachments/assets/c56364e4-e0f8-4f03-9c49-db51a203158c) |
| ```"pna_nic"``` | ![image](https://github.com/user-attachments/assets/b5e15159-e080-4f67-9b28-712e447b0df5)  | ![image](https://github.com/user-attachments/assets/a85a3d49-e9d2-49d9-97b2-6f33266a0cf5)|
| ```'pna_nic'```  | ![image](https://github.com/user-attachments/assets/b0d71ac6-0c1d-4cf5-b94d-3c9844a9de98) | ![image](https://github.com/user-attachments/assets/ff7af775-bc21-4dbb-871e-e0cd22c0b21e)|

Reply:
@rupesh-chiluka-marvell Could you adjust?

@AdarshRawat1 Otherwise, you do not see any other issues, right?

Reply:
Yes

Reply:
I changed the header. Is it ok now?